src/WebJobs.Extensions.DurableTask.Analyzers/Analyzers/ActivityFunction/ArgumentAnalyzer.cs

```diff
@@ -20,9 +20,12 @@ public class ArgumentAnalyzer
         private const string Category = SupportedCategories.Activity;
         public const DiagnosticSeverity Severity = DiagnosticSeverity.Warning;
 
-        public static readonly DiagnosticDescriptor MismatchRule = new DiagnosticDescriptor(DiagnosticId, Title, MismatchMessageFormat, Category, Severity, isEnabledByDefault: true, description: Description);
-        public static readonly DiagnosticDescriptor InputNotUsedRule = new DiagnosticDescriptor(DiagnosticId, Title, InputNotUsedMessageFormat, Category, Severity, isEnabledByDefault: true, description: Description);
-        public static readonly DiagnosticDescriptor InvalidNullRule = new DiagnosticDescriptor(DiagnosticId, Title, InvalidNullMessageFormat, Category, Severity, isEnabledByDefault: true, description: Description);
+        public static readonly DiagnosticDescriptor MismatchRule = new DiagnosticDescriptor(DiagnosticId, Title, MismatchMessageFormat, Category, Severity, isEnabledByDefault: true, description: Description,
+            customTags: WellKnownDiagnosticTags.CompilationEnd);
```

Comment:
What is this tag used for?

Reply:
As far as I can tell, this tag says that the analyzer should run after compilation finishes. It was the default behavior in older versions of this analyzer library, but now there is a warning stating that you should explicitly specify, so I have added everywhere that is applicable